files/vmcore_dump/vmcore-sysctl.conf

```diff
@@ -0,0 +1,7 @@
+kernel.panic_on_oops=1
```

Comment:
Can we move this file to "files/image_config/kdump"

Reply:
Fixed.